source/overview/vpat.rst

```diff
@@ -0,0 +1,481 @@
+
+- **Level of Support & Supporting Features**: NOT APPLICABLE 
+
+- **Remarks and Explanations**: Functionality described in this requirement is not included in the product.
+
+
+Section 1194.26 Desktop and Portable Computers - Detail 
+-----------------------------------------------------------------
+
+(a) All mechanically operated controls and keys shall comply with 1194.23 (k) (1) through (4).
+
+- **Level of Support & Supporting Features**: NOT APPLICABLE 
+
+- **Remarks and Explanations**: Functionality described in this requirement is not included in the product.
+
+(b) If a product utilizes touchscreens or touch-operated controls, an input method shall be provided that complies with 1194.23 (k) (1) through (4).
+
+- **Level of Support & Supporting Features**: NOT APPLICABLE 
+
+- **Remarks and Explanations**: Functionality described in this requirement is not included in the product.
+
+(c) When biometric forms of user identification or control are used, an alternative form of identification or activation, which does not require the user to possess particular biological characteristics, shall also be provided.
+
+- **Level of Support & Supporting Features**: NOT APPLICABLE 
+
+- **Remarks and Explanations**: Functionality described in this requirement is not included in the product.
+
+(d) Where provided, at least one of each type of expansion slots, ports and connectors shall comply with publicly available industry standards
+
+- **Level of Support & Supporting Features**: NOT APPLICABLE 
+
+- **Remarks and Explanations**: Functionality described in this requirement is not included in the product.
+
+
+Section 1194.31 Functional Performance Criteria – Detail
+-----------------------------------------------------------------
+
+(a) At least one mode of operation and information retrieval that does not require user vision shall be provided, or support for Assistive Technology used by people who are blind or visually impaired shall be provided.
+
+- **Level of Support & Supporting Features**: TBD
+
+- **Remarks and Explanations**: TBD
+
+(b) At least one mode of operation and information retrieval that does not require visual acuity greater than 20/70 shall be provided in audio and enlarged print output working together or independently, or support for Assistive Technology used by people who are visually impaired shall be provided.
+
+- **Level of Support & Supporting Features**: SUPPORTED 
+
+- **Remarks and Explanations**: Increasing Zoom level in Mattermost web app can be used to fulfill this requirement. 
+
+(c) At least one mode of operation and information retrieval that does not require user hearing shall be provided, or support for Assistive Technology used by people who are deaf or hard of hearing shall be provided
```

Comment:
Supported, we have a visual clue for anything audible like desktop notifications.